### PR TITLE
Created and added a logo

### PR DESCRIPTION
**Pull request by Adeyomola**
Hello @euanwm I created and added a logo to the index.html and search.html files. Check to see if you like the logo. Thank you.

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -28,13 +23,18 @@
...
-    <center><h1>OpenWeightlifting</h1></center>
-    <div id="root"></div>
-    <!--
+  <title>OpenWeightlifting</title>
+</head>
+
+<body>
+  <noscript>You need to enable JavaScript to run this app.</noscript>
+  <center>
+    <img src="./OWL-logo.png" alt="OWL-logo" width="80" height="40">
```
Comment (euanwm) [inline client/public/index.html:32]: Width="200", height="100" looks good as a main banner.

File: client/public/index.html [hunk 2]
```diff
@@ -28,13 +23,18 @@
...
+    <h3 style="margin: 0; font-family: calibri;">OpenWeightlifting</h3>
```
Comment (euanwm) [inline client/public/index.html:33]: This line can be removed.

**Review by euanwm — COMMENTED** [review]
1 minor change and 1 removal

**Comment by Adeyomola** [comment]
Hello @euanwm I've made the suggested adjustments.

**Pull request merged**